### PR TITLE
ThinkscapeLimited.ZeeDrive version 48.1.0.0

### PR DESCRIPTION
**Pull request by trappsa**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92516)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @trappsa,

I wasn't able to find the combination of switches to have this package install silently. I found the publisher's [silent install instructions](https://www.thinkscape.com/Map-Network-Drives-To-Office-365-OneDrive) (page covers everything, search for "silent"), including "/qn /i ZeeDrive COMMAND=Install", but all attempts resulted in an attended installation. Can you add the silent install switches?

File: manifests/t/ThinkscapeLimited/ZeeDrive/48.1.0.0/ThinkscapeLimited.ZeeDrive.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ThinkscapeLimited.ZeeDrive
+PackageVersion: 48.1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/t/ThinkscapeLimited/ZeeDrive/48.1.0.0/ThinkscapeLimited.ZeeDrive.installer.yaml:8]:
```suggestion
  InstallerType: exe
  InstallerSwitches:
    Silent: Command=Install
    SilentWithProgress: Command=Install
    Upgrade: Command=Upgrade
  InstallerSuccessCodes:
  - 10
  ExpectedReturnCodes:
  - InstallerReturnCode: 11
    ReturnResponse: rebootRequiredToFinish
```

Please note, however, that this application doesn't put any entries into the ARP table so there is no way for winget to detect it once it is installed. Additionally, the only way to launch or configure the application after a silent install is to use the exe file with command line parameters. Therefore, this package probably isn't a good candidate to add to the repo at all.

**Comment by trappsa** [comment]
Thanks considering! You can reject it and not include it. Scott TrappOn Jan 4, 2023, at 9:53 PM, Kaleb Luedtke ***@***.***> wrote:﻿
@Trenly commented on this pull request.



In manifests/t/ThinkscapeLimited/ZeeDrive/48.1.0.0/ThinkscapeLimited.ZeeDrive.installer.yaml:
> @@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ThinkscapeLimited.ZeeDrive
+PackageVersion: 48.1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: exe

⬇️ Suggested change
-  InstallerType: exe
+  InstallerType: exe
+  InstallerSwitches:
+    Silent: Command=Install
+    SilentWithProgress: Command=Install
+    Upgrade: Command=Upgrade
+  InstallerSuccessCodes:
+  - 10
+  ExpectedReturnCodes:
+  - InstallerReturnCode: 11
+    ReturnResponse: rebootRequiredToFinish


Please note, however, that this application doesn't put any entries into the ARP table so there is no way for winget to detect it once it is installed. Additionally, the only way to launch or configure the application after a silent install is to use the exe file with command line parameters. Therefore, this package probably isn't a good candidate to add to the repo at all.

**Comment by stephengillie** [comment]
> Thanks considering! You can reject it and not include it.

**Pull request closed without merging**